monai/transforms/post/array.py

```diff
@@ -179,6 +187,13 @@ def __call__(
         if threshold_values or self.threshold_values:
             img = img >= (self.logit_thresh if logit_thresh is None else logit_thresh)
 
+        rounding = self.rounding if rounding is None else rounding
+        if rounding is not None:
+            if rounding == "torchrounding":
```

Comment:
could use `look_up_options` here

Reply:
Sure, updated in the latest commit.
Thanks.